oap-server/server-starter/src/main/resources/ui-initialized-templates/virtual_mq/virtual-mq-service.json

```diff
@@ -378,7 +382,41 @@
       "name": "Virtual-MQ-Service",
       "id": "Virtual-MQ-Service",
       "isRoot": false,
-      "isDefault": true
+      "isDefault": true,
+      "expressionsConfig": [
+        {
+          "unit": "calls / min",
+          "label": "Consume Traffic"
+        },
+        {
+          "unit": "ms",
+          "label": "Latency Percentile"
```

Comment:
The topology node metric widget shows a group single value, `Percentile` is hard to show as single value. For example in `virtual-database-service.json`:
```
      "expressions": [
        "avg(database_access_resp_time)",
        "avg(database_access_sla)/100",
        "avg(database_access_cpm)"
      ],
      "expressionsConfig": [
        {
          "unit": "ms",
          "label": "Latency"
        },
        {
          "unit": "%",
          "label": "Successful Rate"
        },
        {
          "unit": "calls / min",
          "label": "Traffic"
        }
      ]
```
<img width="505" alt="image" src="https://github.com/apache/skywalking/assets/16773043/a9dae5b5-08de-4f48-b626-25329e1461d0">

Just pick a few representative metrics and expressions, not require all.